### DISCUSSION TITLE
Bem-vindo(a) ao Fórum de Discussões da OBSAT-MCTI

### DISCUSSION THREAD

**Discussion by rafaelaroca** [OBSAT-MCTI/OBSAT-MCTI#1] [Anúncios]
👋 Bem-vindo!

Estamos usando discussões como um lugar para nos conectar com outros membros da nossa comunidade. Esperamos que você:

- Faça perguntas que você está pensando.
- Compartilhe ideias.
- Envolva-se com outros membros da comunidade.
- Bem-vindos aos outros e têm a mente aberta. Lembre-se que esta é uma comunidade que construir juntos 💪.

Para começar, comente abaixo com uma introdução de si mesmo e conte-nos sobre o que você faz com essa comunidade.

**Comment by marisanobrega** [comment]
Olá, boa noite....
Sou professora de Tecnologia e Inovação da Escola Olímpio Catão. As duas equipes estão escrevendo o projeto para fase 2. Tá um pouco difícil ajudá-los na parte técnica...tem alguma equipe disposta a ajudar?
